user/apps.md

```diff
@@ -292,6 +292,16 @@ By ThoughtWorks Inc.
 * [app store](https://itunes.apple.com/us/app/ccmenu/id603117688?mt=12&ign-mpt=uo%3D4)
 * [tutorial](/user/cc-menu/)
 
+![Travis CI in Screensaver Ninja with Custom CSS](/images/apps/screensaver-ninja.gif){:.app}
```

Comment:
I can change it, but that's not true for CCMenu, not sure about others, but the next entry, which is for Linux, seems to also have the screenshot above the heading. Do you want me to still change it?